dev/Installer/src/ProjectReunionInstall.vcxproj

```diff
@@ -0,0 +1,157 @@
+      <PreprocessorDefinitions>_CONSOLE;WIN32_LEAN_AND_MEAN;WINRT_LEAN_AND_MEAN;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <WarningLevel>Level4</WarningLevel>
+      <AdditionalOptions>%(AdditionalOptions) /permissive- /bigobj</AdditionalOptions>
+    </ClCompile>
+  </ItemDefinitionGroup>
+  <ItemDefinitionGroup Condition="'$(Configuration)'=='Debug'">
+    <ClCompile>
+      <Optimization>Disabled</Optimization>
+      <PreprocessorDefinitions>_DEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+    </ClCompile>
+    <Link>
+      <SubSystem>Console</SubSystem>
+      <GenerateWindowsMetadata>false</GenerateWindowsMetadata>
+      <AdditionalDependencies Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">onecore.lib;onecoreuap.lib;%(AdditionalDependencies)</AdditionalDependencies>
+    </Link>
+  </ItemDefinitionGroup>
+  <ItemDefinitionGroup Condition="'$(Platform)'=='Win32'">
+    <ClCompile>
+      <PreprocessorDefinitions>WIN32;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+    </ClCompile>
+    <Link>
+      <AdditionalDependencies Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">onecore.lib;onecoreuap.lib;%(AdditionalDependencies)</AdditionalDependencies>
+    </Link>
+  </ItemDefinitionGroup>
+  <ItemDefinitionGroup Condition="'$(Configuration)'=='Release'">
+    <ClCompile>
+      <Optimization>MaxSpeed</Optimization>
+      <FunctionLevelLinking>true</FunctionLevelLinking>
+      <IntrinsicFunctions>true</IntrinsicFunctions>
+      <PreprocessorDefinitions>NDEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <RuntimeLibrary Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">MultiThreaded</RuntimeLibrary>
+    </ClCompile>
+    <Link>
+      <SubSystem>Console</SubSystem>
+      <EnableCOMDATFolding>true</EnableCOMDATFolding>
+      <OptimizeReferences>true</OptimizeReferences>
+      <GenerateWindowsMetadata>false</GenerateWindowsMetadata>
+      <AdditionalDependencies Condition="'$(Configuration)|$(Platform)'=='Release|x64'">onecore.lib;onecoreuap.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">onecore.lib;onecoreuap.lib;%(AdditionalDependencies)</AdditionalDependencies>
+    </Link>
+  </ItemDefinitionGroup>
+  <ItemGroup>
+    <ClInclude Include="help.h" />
+    <ClInclude Include="install.h" />
+    <ClInclude Include="packages.h" />
+    <ClInclude Include="pch.h" />
+    <ClInclude Include="project_reunion_definitions.h" />
+    <ClInclude Include="resource.h" />
+  </ItemGroup>
+  <ItemGroup>
+    <ClCompile Include="help.cpp" />
+    <ClCompile Include="install.cpp" />
+    <ClCompile Include="main.cpp" />
+    <ClCompile Include="pch.cpp">
+      <PrecompiledHeader>Create</PrecompiledHeader>
+    </ClCompile>
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="packages.config" />
+    <None Include="Packages\x64\Microsoft.ProjectReunion.0.1.appx" />
```

Comment:
The extension for Windows app packages is `.msix` - please fix *all* instances, in code or in solution configuration.

Reply:
The place for that change is probably not here.  I queried a bunch of people over our usage of .appx and got a <shrug> but I apparently didn't poll you at the time. :-)  Can you file an issue on this?  It has engineering system costs so it isn't a trivial change and shouldn't be dirven by David's PR (IMO).

Reply:
> You can fix this after we finish fixing this issue:  https://github.com/microsoft/ProjectReunion/issues/447

Reply:
> The extension for Windows app packages is `.msix` - please fix _all_ instances, in code or in solution configuration.

That's the exact filename in the nuget distribution for released version 0.1, so that's what I have as the default file extension for the frameworks.  Though that said, it doesn't matter what the extension is for deployment to consume it (and an extension isn't specified when copying from memory to temp file), so there isn't need to have any specific extension.

However, the entries here in the vcxproj are actually an artifact from an earlier version of the installer and can be removed enitrely, so I'll do that. The ".appx" specification in the project_reunion_definitions.h file I will leave alone for now until the frameworks in the nuget are changed. If we have these frameworks on a release share in .msix form I can update the paths to point there by default.

Reply:
.msix is used everywhere in this project now.